examples/training_without_local_storage.ipynb

```diff
@@ -0,0 +1,1005 @@
+    ")\n",
+    "eval_dataloader = torch.utils.data.DataLoader(\n",
+    "    dataset=eval_dataset,\n",
+    "    batch_size=batch_size,\n",
+    ")\n",
+    "\n",
+    "cloud_logger = ObjectStoreLogger(\n",
+    "    object_store_cls=S3ObjectStore,\n",
+    "    # Keyword arguments passed to the S3ObjectStore constructor\n",
+    "    object_store_kwargs={\n",
+    "        'bucket': s3_bucket_name,\n",
+    "        'prefix': bucket_prefix,\n",
+    "    },\n",
+    "    use_procs=False,\n",
+    ")\n",
+    "\n",
+    "trainer = Trainer(\n",
+    "    model=model,\n",
+    "    max_duration='2ep',\n",
+    "    # Make training fast: Terminate each epoch after 5 batches\n",
+    "    train_subset_num_batches=5,\n",
+    "    optimizers=optimizer,\n",
+    "    train_dataloader=train_dataloader,\n",
+    "    eval_dataloader=eval_dataloader,\n",
+    "    # Make evaluation fast: Evaluate only on five batches\n",
+    "    eval_subset_num_batches=5,\n",
+    "    loggers=[\n",
+    "        cloud_logger,\n",
+    "        TensorboardLogger(log_dir=tensorboard_log_dir, flush_interval=1),\n",
+    "    ],\n",
+    "    save_folder=checkpoint_dir,\n",
+    "    save_interval=\"1ba\",\n",
+    "    # Because we are uploading checkpoints to the cloud,\n",
+    "    # delete them locally to save disk space!\n",
+    "    save_num_checkpoints_to_keep=0,\n",
+    "    run_name=run_name,\n",
+    ")\n",
+    "\n",
+    "########\n",
+    "# Train!\n",
+    "########\n",
+    "\n",
+    "print(\"Training from the beginning\")\n",
+    "trainer.fit()\n",
+    "trainer.close()\n",
+    "\n",
+    "\n",
+    "#################################\n",
+    "# Train again (from a checkpoint)\n",
+    "#################################\n",
+    "\n",
+    "trainer = Trainer(\n",
+    "    model=model,\n",
+    "    max_duration='4ep',\n",
+    "    load_path=f'{run_name}/checkpoints/latest-rank0',\n",
+    "    load_object_store=cloud_logger,\n",
+    "    train_subset_num_batches=5,\n",
+    "    optimizers=optimizer,\n",
+    "    train_dataloader=train_dataloader,\n",
+    "    eval_dataloader=eval_dataloader,\n",
+    "    eval_subset_num_batches=5,\n",
+    "    loggers=[\n",
+    "        cloud_logger,\n",
+    "        TensorboardLogger(log_dir=tensorboard_log_dir, flush_interval=1),\n",
+    "    ],\n",
+    "    save_folder=checkpoint_dir,\n",
+    "    save_interval=\"1ba\",\n",
+    "    save_num_checkpoints_to_keep=0,\n",
+    "    run_name=run_name,\n",
+    ")\n",
+    "\n",
+    "print(\"Training from a checkpoint\")\n",
+    "trainer.fit()\n",
+    "trainer.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "###################\n",
+    "# Visualize Results\n",
+    "###################\n",
+    "\n",
+    "import tensorboard.notebook\n",
+    "%load_ext tensorboard\n",
+    "\n",
+    "%tensorboard --logdir s3://{s3_bucket_name}/{bucket_prefix}"
```

Comment:
I couldn't get any of the tensorboard commands to work so maybe don't listen to me, but I think maybe this should be `

E.g., my printed s3 bucket contents don't contain and tfevents files under my bucket_prefix:

```
local-storage-notebook-test/.credentials_validated_successfully
local-storage-notebook-test/cloud_training_run/checkpoints/ep0-ba1-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep0-ba2-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep0-ba3-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep0-ba4-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep0-ba5-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep1-ba10-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep1-ba6-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep1-ba7-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep1-ba8-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/ep1-ba9-rank0
local-storage-notebook-test/cloud_training_run/checkpoints/latest-rank0.symlink
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093698.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093700.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093701.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093703.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093704.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093709.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093711.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093712.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093714.Daviss-MacBook-Pro.local.81197-0
local-storage-notebook-test/tensorboard_logs/cloud_training_run/events.out.tfevents.1660093715.Daviss-MacBook-Pro.local.81197-0
local-storage
```

Reply:
Good catch; updated to include the postfix `tensorboard_logs`. Re: stuff not showing up, most likely a credentials error. I had better luck on the `mosaicml-internal-checkpoints-test` bucket.